src/test/owr/OptimisticWithdrawalRecipient.t.sol

```diff
@@ -70,6 +82,45 @@ contract OptimisticWithdrawalRecipientTest is OWRTestHelper, Test {
     assertEq(wtrancheThreshold, ETH_STAKE, "invalid eth tranche threshold");
   }
 
+  function testOWRPectraInitialization() public {
+    assertTrue(owrETHPectra.initialized());
+    assertEq(owrETHPectra.owner(), address(this));
+  }
+
+  function testReinitialization() public {
+    assertTrue(owrETHPectra.initialized());
+    vm.expectRevert(OptimisticWithdrawalRecipientPectra.InvalidRequest_Initialize.selector);
+    owrETHPectra.initialize(address(this));
+  }
+
+  function testInitiateWithdrawal(uint8 amount) public {
+    vm.assume(amount >= 100 && amount <= 100000);
+    address _user = vm.addr(0x1);
+    vm.deal(_user, type(uint256).max);
+
+    bytes memory pubkey = new bytes(48);
+    for (uint256 i = 0; i < 48; i++) {
+        pubkey[i] = bytes1(0xAB);
+    }
+
+    vm.startPrank(_user);
+    vm.expectRevert(0x82b42900); // Unauthorized
+    owrETHPectra.requestPrincipalWithdrawal{value: 1 ether}(pubkey, amount);
+
+    vm.stopPrank();
+    vm.startPrank(address(this));
+    owrETHPectra.grantRoles(_user, 1111);
+    owrETHPectra.grantRoles(_user, 2222);
+
+    vm.stopPrank();
+    vm.startPrank(_user);
+    owrETHPectra.requestRewardsWithdrawal{value: 1 ether}(pubkey, amount);
+    vm.stopPrank();
+
+    assertGt(withdrawalMock.receivedAmount(), 0);
```

Comment:
Seems like a bit of an easy test to pass considering amount is from 100 to 100000?